go/vt/vtorc/inst/instance_dao.go

```diff
@@ -156,6 +145,14 @@ func logReadTopologyInstanceError(tabletAlias string, hint string, err error) er
 	return fmt.Errorf(msg)
 }
 
+// RegisterStats registers stats from the inst package
+func RegisterStats() {
+	stats.NewGaugeFunc("ErrantGtidTabletCount", "Number of tablets with errant GTIDs", func() int64 {
```

Comment:
If it is a gauge then I would prefer the metric name not to contain `count`. 
`Number of tablets with errant GTIDs` -> `Current tablets having errant GTIDs`

Reply:
Well, it is the count of the tablets that have errant GTIDs. Its just that the count can go up and down...